pkg/actuators/machine/reconciler.go

```diff
@@ -20,6 +21,8 @@ const (
 	requeueAfterSeconds      = 20
 	requeueAfterFatalSeconds = 180
 	masterLabel              = "node-role.kubernetes.io/master"
+	// CacheTTL is duration of time to store the vm ip in cache
+	CacheTTL = time.Duration(15) * time.Minute
```

Comment:
why is this 15mins? we need to understand what's the lease period for the dhcp IP address and set the tll accordingly

Reply:
and this can be moved to that new file caching.go